packages/video_player/video_player_android/android/build.gradle

```diff
@@ -47,12 +47,23 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
     }
 
+    // Gradle 6.x workaround for
+    // see "Selecting the appropriate flavor"
+    // https://github.com/google/guava/releases/tag/v32.1.0
+    dependencies.constraints {
+        implementation("com.google.guava:guava") {
+            attributes {
+                attribute(Attribute.of("org.gradle.jvm.environment", String), "android")
+            }
+        }
+    }
```

Comment:
gradle 7+ handles this fine with 32.1.2+
gradle 6 needs workaround described at
[google/guava@v32.1.0 (release)](https://github.com/google/guava/releases/tag/v32.1.0)
https://github.com/google/guava/issues/6612#issuecomment-1614992368

Reply:
welp, it's fixed now for video_player_android, but fails somehow for project :espresso with 30.1.1-android (all_packages)
still investigating

Reply:
dropped commit